### PR TITLE
Add new option to Japan.md, and some story to follow it

### PR DESCRIPTION
**Pull request by coralvanda**
Not sure @casey-collab can merge this.  It shows a deletion, but I don't recall making one.

### CODE REVIEW

File: english/dream/fly/asia/japan/japan.md [hunk 1]
```diff
@@ -2,4 +2,6 @@ You have decided to leave India and fly to Japan.
 
 You land in Tokyo Haneda Airport in the morning.
 
-What is your first stop in Japan?
\ No newline at end of file
+What is your first stop in Japan?
+
+[Board a train](english/dream/fly/asia/japan/train.md) heading to Mount Fuji.
```
Comment (casey-collab) [inline english/dream/fly/asia/japan/japan.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/fly/asia/japan/train/train.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+You board the next train heading to Mount Fuji, and arrive much faster than
```
Comment (casey-collab) [inline english/dream/fly/asia/japan/train/train.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by coralvanda** [comment]
@casey-collab try it now.

File: english/dream/fly/asia/japan/japan.md [hunk 3]
```diff
@@ -2,4 +2,6 @@ You have decided to leave India and fly to Japan.
 
 You land in Tokyo Haneda Airport in the morning.
 
-What is your first stop in Japan?
\ No newline at end of file
+What is your first stop in Japan?
+
+[Board a train](../train/train.md) heading to Mount Fuji.
```
Comment (casey-collab) [inline english/dream/fly/asia/japan/japan.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/fly/asia/japan/train/train.md (hunk 2 above)
Comment (casey-collab) [inline english/dream/fly/asia/japan/train/train.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by coralvanda** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**